src/test/kotlin/pt/up/fe/ni/website/backend/utils/documentation/payloadschemas/model/PayloadProject.kt

```diff
@@ -34,6 +34,12 @@ class PayloadProject : ModelDocumentation(
         DocumentedJSONField(
             "targetAudience",
             "Information about the target audience",
+            JsonFieldType.STRING,
+            optional = true
```

Comment:
If this indicates whether the field is required or not, currently it is. We could discuss placing it as optional though, we talked about every project having a target audience, but currently the mandatory information would be just `title`, `description` and `targetAudience`. In that list I really do not think it is as important as the others and that a project could be well described without it.

Reply:
You're right, I'll change that! We can re-discuss that in the next meeting too